### DISCUSSION TITLE
Reliability and credibility of Stripe

### DISCUSSION THREAD

**Discussion by KarlKeu00** [pretix/pretix#4019] [Q&A]
Hello everyone,
I have a question and wanted to know if this has happened to any of you.
Months ago, I set up a Stripe account for our association and tested the transfers with numerous test payments. I have entered all the necessary data in our e.V. company account.
We have been online with the shop for our members for a few days now and have already been able to "successfully" process payments via Stripe. Unfortunately, Stripe is now accusing us of having "detected a high number of unauthorised payments on your account". As a result, the 10 successful payments will not be paid out to us.
Yesterday we also received an email that our account will be deleted by the end of the month.
Has anyone had similar experiences with Stripe? Unfortunately, the support is very slow to respond and only answers half of the questions (if at all).
Many thanks for your help.

**Comment by pc-coholic** [comment]
That sucks - so sorry to hear this :-(

Unfortunately, Stripe has not only hit the "too big to fail", but also the "too big to talk to" threshold. So pretty much the same as most of the big players in the mass-market.

While not having experienced it myself or having had a direct customer who experienced this, I have indeed read this happening. At this point, it probably doesn't matter if you are an e.V. or a GmbH: if someone (or something) at Stripe has set their minds to "fraudulent", your account is gone and you can be happy, if you get all the funds in the account.

So far, I haven't seen anyone getting a decent answer as to why things like this happen. But having excessive payments and refunds seem to play into this. So testing in production and refunding those payments, all from the same IP is apparently not a great look.

But in the end, it's everyone's guess: perhaps the board of directors just happen to have a member whose name is just a bit too similar to someone on a embargo list...

You can try to name and shame them on social media. If you are big and well known enough, you might be successful. You could even try sueing - but since all contracts are "at will", they are under no obligation to actually serve you...

While Stripe has a lot of comfort for a merchant looking to set up an account, I would probably not spend too much energy on trying to resolve the situation: the chances of getting someone's attention are very slim.

My recommendation: move on to another payment processor, possibly a smaller one where you still have personalized onboarding and can reach people on the phone.

Or find someone who has a lot of pull at Stripe (like a merchant processing large amounts) and ask them to involve their partner/account manager.

If you are a hosted pretix.eu customer, you can also reach out to support - while we do not see anything more than you, we can *try* to involve a partner manager. That being said: even that would take some time and more often than not is a one way street where we are not kept inside the loop.

**Reply by pc-coholic** [reply]
Actually, after thinking some more, I do remember one case where an account was closed and money withheld/refunded...

The reasoning at the time however was not "unauthorized payments" - the account was just closed.

Some digging revealed, that Stripe requested additional information on the account owners/board of directors, including copies of ID cards and the merchant just ignored the messages, saying they never got such a notice...

**Reply by KarlKeu00** [reply]
> Unfortunately, Stripe has not only hit the "too big to fail", but also the "too big to talk to" threshold. So pretty much the same as most of the big players in the mass-market.

This is a sentence that unfortunately applies to too many companies. It's a shame that this has already become a trend.

In fact, we were asked if we could upload proof of our existence. However, the extract from the german commercial register was apparently rejected/ignored... The alternative would have been, among other things, bank statements for the last 6 months for the business account _or_ a certificate of trade tax or VAT registration. Now the "or" has been changed to an "and".

**Reply by pc-coholic** [reply]
Stripe follows a concept of "least friction" for the merchant - that's also the reason why you can start taking payments before going through proper KYC...

Once you start taking payments, they will ask for a minimum set of documents to validate your company/identity/dog/shoe size. Only if the previously submitted documents are not deemed enough to validate the "thing", the previously optional documents will become compulsory...

So this is - for Stripe - the expected behavior...

**Reply by KarlKeu00** [reply]
Maybe it would help if Pretix would write a little note in the Stripe plugin? Maybe it would help if Pretix would write a little note in the Stripe plugin? At least that tests in test mode are not meaningful and only a transfer in live mode triggers a verification of the account?
By the way: All SEPA direct debits were marked as failed and simply not debited. No transaction fees were charged either. For credit card payments, I can't yet say whether the fees will be refunded - support apparently doesn't know.

**Reply by raphaelm** [reply]
I don't think we'll add a warning about this in pretix.

All payment providers need to do risk assessments, and all payment providers larger than some threshold sometimes get these wrong, and all of them are hard to reason with about that. I don't think Stripe is special here, and in general, Stripe is one of the easier ones to get an account with.

**Comment by KarlKeu00** [comment]
Small update:

Even after we have submitted official documents from the commercial register and the tax authorities, we are still categorised as an "unacceptable risk". There is no explanation as to why, but there is probably nothing we can do about it.

Fortunately, the SEPA direct debits were already cancelled on day 1 without any transaction fees. However, nobody was able to give me a concrete answer regarding the credit card payments. There are probably several options. The first is that everything is refunded and the transaction fees are debited from our account, the second is that everything is refunded as long as we can finance the fees from the revenue (= one customer does not receive a refund) and the third is that nothing is refunded. Nobody could really tell me.
